repos/spack_repo/builtin/packages/netcdf_c/package.py

```diff
@@ -143,6 +143,20 @@ class NetcdfC(CMakePackage, AutotoolsPackage):
         when="@4.9.0:4.9.2",
     )
 
+    # https://github.com/Unidata/netcdf-c/issues/3199
+    patch("cmakelists_mpi_symbols.patch", when="build_system=cmake")
+
+    def patch(self):
+        """Fix bad code in ncgen/CMakeLists.txt that removes
+        the rpath for dependencies like hdf5."""
+        if self.spec.satisfies("build_system=cmake"):
+            filter_file(
+                "SET(CMAKE_INSTALL_RPATH_USE_LINK_PATH FALSE)",
+                "#SET(CMAKE_INSTALL_RPATH_USE_LINK_PATH FALSE)",
```

Comment:
If you want this enabled, I'd explicitly set this to true, i.e.

`set(CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE)`

Otherwise this could still be false unexpectedly. This doesn't stop other places in the code from changing the setting, but explicitly setting it here guarantees you'll be getting the expected settings for the targets you care about.

Reply:
Good point, happy to make this change.